### PR TITLE
Use logging functions in import code.

### PR DESCRIPTION
**Pull request by portnov**
This is basically a s/print/debug/. I tried to choose severity level according to context.

@zeffii  please review.

### CODE REVIEW

File: utils/sv_IO_panel_tools.py [hunk 1]
```diff
@@ -218,17 +219,17 @@ def create_dict_of_tree(ng, skip_set={}, selected=False):
         for k, v in node.items():
 
             if not isinstance(v, (float, int, str)):
-                print('//')
-                print(node.name, ' -> property:', k, type(v))
+                debug('//')
```
Comment (zeffii) [inline utils/sv_IO_panel_tools.py]: i'm a little torn on this..i suppose if I really need to see this message i'd switch on debug mode..

Comment (zeffii) [reply utils/sv_IO_panel_tools.py]: (updated my comment)

File: utils/sv_IO_panel_tools.py [hunk 2]
```diff
@@ -674,8 +673,8 @@ def generate_layout(fullpath, nodes_json):
         # this will handle both scenarios
         if isinstance(nodes_json, str):
             nodes_json = json.loads(nodes_json)
-            print('==== loading monad ====')
-        print('#' * 12, nodes_json['export_version'])
+            debug('==== loading monad ====')
+        print(('#' * 12) + nodes_json['export_version'])
```
Comment (zeffii) [inline utils/sv_IO_panel_tools.py]: info ? instead of print?

**Comment by zeffii** [comment]
nothing jumps out at me that I don't agree with.

File: utils/sv_IO_panel_tools.py (hunk 2 above)
Comment (portnov) [reply utils/sv_IO_panel_tools.py]: ok, just overlooked

**Pull request merged**